cocos/rendering/legacy/index.jsb.ts

```diff
@@ -53,7 +53,7 @@ import { RenderTexture } from '../../asset/assets/render-texture';
 
 export function createDefaultPipeline (): ForwardPipeline {
     const rppl = new ForwardPipeline();
-    rppl.initialize({ flows: [] });
+    //rppl.initialize({ flows: [] });
```

Comment:
Why comment it?

Reply:
It will be initialized again later in the process, because openharmony doesn't support proxies, so initializing it twice can be problematic

Reply:
Can we add oh macro around `rppl.initialize({ flows: [] });`?